packages/jmix-react-ide-toolbox/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@haulmont/react-ide-toolbox",
+  "name": "@haulmont/jmix-react-ide-toolbox",
```

Comment:
please revert jmix- prefix (as this package could be potentially used without jmix)